hbase-mapreduce/src/main/java/org/apache/hadoop/hbase/mapreduce/RowCounter.java

```diff
@@ -105,9 +158,11 @@ public void map(ImmutableBytesWritable row, Result values, Context context) thro
    * @throws IOException When setting up the job fails.
    */
   public Job createSubmittableJob(Configuration conf) throws IOException {
+    conf.setBoolean(OPT_COUNT_DELETE_MARKERS, this.countDeleteMarkers);
```

Comment:
nit: move this where other conf are being set, seems placed a little off

Reply:
I mean place it somewhere below, may be after block 
```
if (this.expectedCount >= 0) {
      conf.setLong(EXPECTED_COUNT_KEY, this.expectedCount);
    }
```

Reply:
This is not possible as we need to create a MR Job object using the conf. So technically, all required properties must be set before using the conf for getting a Job object.